dotcom-rendering/src/components/ManyNewsletterSignUp.importable.tsx

```diff
@@ -189,6 +190,9 @@ export const ManyNewsletterSignUp = () => {
 	);
 
 	const removeAll = useCallback(() => {
+		if (status === 'Success') {
+			setHideForm(true);
+		}
```

Comment:
Is this what the `removeAll` method is for? I can see on `L146` that if status is 'Success', `userCanInteract` will always be `false`.

I think `hideForm` should be handled in its own hook.